### PR TITLE
Increase RPC and provider timeout constants

### LINKED ISSUE #460: Increase Provider and RPC timeouts to 15 seconds (currently 5 seconds)
https://github.com/api3dao/airseeker/blob/main/src/constants.ts#L3

### PR DESCRIPTION
**Pull request by aquarat**
Closes #460

I'm running this locally with the production config and so far it looks really good; very few gateway failures.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { ethers } from 'ethers';
 
-export const GATEWAY_TIMEOUT_MS = 5_000;
-export const PROVIDER_TIMEOUT_MS = 5_000;
+export const GATEWAY_TIMEOUT_MS = 15_000;
+export const PROVIDER_TIMEOUT_MS = 15_000;
```
Comment (acenolaza) [inline src/constants.ts:4]: So not only gateways but RPCs are also timing out with 5secs?

**Review by acenolaza — APPROVED** [review]
LGTM 👍🏻 Merge it whenever you feel comfortable that it doesn't generate any side effects 💪🏻

File: src/constants.ts (hunk 1 above)
Comment (acenolaza) [reply src/constants.ts:4]: nvm, I just saw this comment 😄 https://api3workspace.slack.com/archives/C05MXJF7831/p1692352864842519?thread_ts=1692352070.363849&cid=C05MXJF7831

File: src/constants.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 export const RANDOM_BACKOFF_MIN_MS = 0;
-export const RANDOM_BACKOFF_MAX_MS = 2_500;
+export const RANDOM_BACKOFF_MAX_MS = 15_000;
```
Comment (bdrhn9) [inline src/constants.ts]: I think that 15 secs backoff is too large. I'd leave it as is. Other than that it LGTM 🔥

**Review by bdrhn9 — APPROVED** [review]

**Pull request merged**